website/project/metadata/osf-preregistration.json

```diff
@@ -144,7 +144,7 @@
                 },
                 {
                     "text": "Registration following analysis of the data",
-                    "tooltip": "As of the date of submission, you have accessed and analyzed some of the data relevant to the research plan. This includes preliminary analysis of variables, calculation of descriptive statistics, and observation of data distributions. Please contact us (prereg@cos.io) and we will be happy to help you."
+                    "tooltip": "As of the date of submission, you have accessed and analyzed some of the data relevant to the research plan. This includes preliminary analysis of variables, calculation of descriptive statistics, and observation of data distributions. Please see https://cos.io/prereg for more information."
```

Comment:
@Johnetordoff Will these changes require a migration?

Reply:
Oh that's my bad. I pushed a migration to release/next-platform but not to this branch because I thought this PR already had an `ensure_schemas` migration. This PR will need this: https://github.com/CenterForOpenScience/osf.io/commit/8cbe44bb5ea1f9940e21a37c5ddc4fc011b828d1